ledger/sandbox/src/main/scala/db/migration/postgres/V2_1__Rebuild_Acs.scala

```diff
@@ -473,7 +473,7 @@ class V2_1__Rebuild_Acs extends BaseJavaMigration {
     rejectionType match {
       case "Inconsistent" => Inconsistent(description)
       case "OutOfQuota" => OutOfQuota(description)
-      case "TimedOut" => TimedOut(description)
```

Comment:
Changing this here doesn't mean that it will actually updated existing rows that were inserted AFTER the V2_1 migration has run. This only runs once on a database if it hasn't been run before.

Reply:
Yes, this change is not very nice. I removed the `TimedOut` type, so I had to deserialize "TimedOut" constants to _something_ here. Again a case where this migration depending on external code/types is problematic.
 
My proposal is to leave this change here, and add a new migration that renames all "TimedOut" to "InvalidLedgerTime".